core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -116,6 +116,9 @@
     private static final OpenOption[] DEFAULT_OPEN_OPTIONS = { TRUNCATE_EXISTING, WRITE, CREATE };
     private static final BiPredicate<Path, BasicFileAttributes> IS_JSON_FILE_PREDICATE = new IsJsonFilePredicate();
 
+    private static final String MANIFEST_SECTION_SEPARATOR = "::";
+    private static final String MANIFEST_ENTRY_KV_SEPARATOR = "\\=\\|\\|\\=";
```

Comment:
@geoand As we will just be using the property, should I simplify these separators to just ":" and "="?

Reply:
Yeah, that makes sense